src/mongocrypt-kms-ctx.c

```diff
@@ -1067,9 +1186,23 @@ _mongocrypt_kms_ctx_init_azure_wrapkey (
    char *request_string;
    bool ret = false;
 
+   if (!kms) {
```

Comment:
Suggest converting into a not-null assertion.

Reply:
Changed as suggested.  Will show up in next commit.